dev/com.ibm.websphere.appserver.features/visibility/public/microProfile-4.0/io.openliberty.microProfile-4.0.feature

```diff
@@ -0,0 +1,26 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
+symbolicName=io.openliberty.microProfile-4.0
+visibility=public
+singleton=true
+IBM-App-ForceRestart: install, \
+ uninstall
+IBM-ShortName: microProfile-4.0
+Subsystem-Version: 7.0.0
+Subsystem-Name: MicroProfile 4.0
+-features=\
+  com.ibm.websphere.appserver.servlet-4.0, \
+  com.ibm.websphere.appserver.cdi-2.0, \
+  com.ibm.websphere.appserver.jaxrs-2.1, \
+  com.ibm.websphere.appserver.jaxrsClient-2.1, \
+  com.ibm.websphere.appserver.jsonb-1.0, \
+  com.ibm.websphere.appserver.jsonp-1.1, \
+  com.ibm.websphere.appserver.mpConfig-1.4, \
```

Comment:
Should be change to mpConfig-2.0 now?

Reply:
We don't have the updated version of smallrye config yet.